src/context.rs

```diff
@@ -1,15 +1,35 @@
-use std::{ops::Deref, sync::OnceLock};
+use dashmap::DashMap;
+use std::{collections::HashMap, ops::Deref, sync::OnceLock};
+use tokio::sync::watch;
 use twilight_http::Client;
+use twilight_model::id::{Id, marker::CommandMarker};
+use twilight_standby::Standby;
 
 pub static CONTEXT: Ref = Ref(OnceLock::new());
 
 #[derive(Debug)]
 pub struct Context {
+    pub command_map: HashMap<Id<CommandMarker>, crate::command::Type>,
```

Comment:
This is not necessary, we could match on the command name. But this is slightly nicer I think. It should(?) be safe